data/presets/building/construction.json

```diff
@@ -1,5 +1,11 @@
 {
     "icon": "maki-building",
+    "fields": [
+        "name",
```

Comment:
I'd recommend adding "construction" field as well as I see it [already being used very frequently](https://taginfo.openstreetmap.org/tags/building=construction#combinations) and the [wiki itself](https://wiki.openstreetmap.org/wiki/Tag:building%3Dconstruction) points towards using this combination to specify the actual type of a building

Reply:
Good call, I've added that one.